### PR TITLE
GAIA-11454: Fix entity contains requests for non-regions requests

### PR DESCRIPTION
**Pull request by HonghuiLu2021**
1. Remove includeHistorical param unless the entity_type=regions.
2. Add verifications for request params
3. black format updated code.

Test:
all unit tests pass.

### CODE REVIEW

File: groclient/client.py [hunk 1]
```diff
@@ -918,7 +900,7 @@ def get_descendant(
             if the `entity_type` is 'regions'. If provided along with `distance`, `distance` will
             take precedence. If not provided, and `distance` not provided, get all descendants.
         include_historical : boolean, optional
-            True by default. If False is specified, regions that only exist in historical data
+            Only applicable to regions. True by default. If False is specified, regions that only exist in historical data
```
Comment (HonghuiLu2021) [inline groclient/client.py:903]: Clarify documentations.

File: groclient/lib.py [hunk 2]
```diff
@@ -618,22 +634,29 @@ def get_geojsons(access_token, api_host, region_id, descendant_level, zoom_level
...
+    access_token,
+    api_host,
+    entity_type,
+    entity_id,
+    distance=None,
+    include_details=True,
+    ancestor_level=None,
+    include_historical=True,
+):
+    url = f'https://{api_host}/v2/{entity_type}/belongs-to'
```
Comment (HonghuiLu2021) [inline groclient/lib.py:659]: Replace with f-string to improve readability.

File: groclient/lib.py [hunk 3]
```diff
@@ -675,7 +704,8 @@ def get_descendant(access_token, api_host, entity_type, entity_id, distance=None
         else:
             params['distance'] = -1
 
-    params['includeHistorical'] = include_historical
+    if entity_type == 'regions':
```
Comment (HonghuiLu2021) [inline groclient/lib.py:707]: This is the fix - metrics and items should not set the `includeHistorical` flag.

File: groclient/lib_test.py [hunk 4]
```diff
@@ -408,74 +438,171 @@ def test_get_ancestor(mock_requests_get, lookup_mocked):
 
 @mock.patch('groclient.lib.lookup')
 @mock.patch('requests.get')
-def test_get_descendant(mock_requests_get, lookup_mocked):
+def test_get_descendant_for_items(mock_requests_get, lookup_mocked):
```
Comment (HonghuiLu2021) [inline groclient/lib_test.py:441]: Create separate tests for each entity type.

File: groclient/lib_test.py [hunk 5]
```diff
@@ -408,74 +438,171 @@ def test_get_ancestor(mock_requests_get, lookup_mocked):
...
 
-    assert lib.get_descendant(MOCK_TOKEN, MOCK_HOST, 'items', 4) == [
+    assert lib.get_descendant(MOCK_TOKEN, MOCK_HOST, ITEMS_ENTIRY, 4) == [
         {'id': 1, 'name': 'item 1', 'contains': [], 'belongsTo': [3], 'definition': 'def1'},
         {'id': 2, 'name': 'item 2', 'contains': [], 'belongsTo': [3], 'definition': 'def2'},
+        {'id': 3, 'name': 'parent', 'contains': [1, 2], 'belongsTo': [4], 'definition': 'def3'},
+    ]
+    mock_requests_get.assert_called_with(
+        f'https://{MOCK_HOST}/v2/{ITEMS_ENTIRY}/contains',
+        params={'ids': [4], 'distance': DESCENDANT_DISTANCE_NAN},
```
Comment (HonghuiLu2021) [inline groclient/lib_test.py:454]: Verify request parameters - no `includeHistorical`.

**Review by John-Desmond — APPROVED** [review]
LGTM

**Review by cn1036 — APPROVED** [review]
code change LGTM. tested locally

**Pull request merged**